src/components/SingleArticlePage.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react"
+import fetchData from "../utils/api"
+import { useParams, useNavigate } from "react-router-dom"
+import { Button } from "react-bootstrap"
+
+
+const SingleArticlePage = () => {
+    const {article_id} = useParams()
+    const [article, setArticle] = useState({})
+    const navigate = useNavigate()
+
+    useEffect(() => {
+        fetchData({endpoint: `/articles/${article_id}`})
+        .then(({article}) => {
+            setArticle(article)
+        })
+    })
```

Comment:
we're now entering error territory where someone could navigate in the page path to '/articles/999999' without using the articles list page at all. Make sure you're throwing some kind of simple 404 message at least when this happens.

Currently all this happens when i navigate to a incorrect article ID:
![image](https://user-images.githubusercontent.com/61753159/192502137-58147d07-a697-40a7-ab2d-3c6fbcd7cafc.png)

Reply:
should i not be covering this in ticket 12?

Reply:
You're right it is in ticket 12 but thanks for doing it now, I still think we should address it now

Reply:
yeah makes sense :)